js/dnd.js

```diff
@@ -137,11 +135,8 @@ module.exports.isMiddle = (domNode, clientX) => {
 module.exports.prepareBookmarkDataFromCompatible = (dataTransfer) => {
   let bookmark = dndData.getDragData(dataTransfer, dragTypes.BOOKMARK)
   if (!bookmark) {
-    const frameProps = dndData.getDragData(dataTransfer, dragTypes.TAB)
-    if (frameProps) {
-      bookmark = siteUtil.getDetailFromFrame(frameProps, siteTags.BOOKMARK)
-      appActions.addSite(bookmark, siteTags.BOOKMARK)
-    }
+    const frame = dndData.getDragData(dataTransfer, dragTypes.TAB)
+    windowActions.onFrameBookmark(frame.get('activeFrameKey'))
```

Comment:
Probably the naming is wrong here, so I fixed it. We send this data from this line https://github.com/brave/browser-laptop/pull/9754/files/e0827f5b092eecfbe915b23838fb9dccc8fdcc5b#diff-51a114fecaac13cd41162f4c956e7508R114 and then get it back here as Immutable Map with this function https://github.com/brave/browser-laptop/blob/26a580bc3148ec427465c3e266e6bd42778da786/js/dndData.js#L11